docs/deploy/index.md

```diff
@@ -428,7 +427,7 @@ kubectl exec $POD_NAME -n $POD_NAMESPACE -- /nginx-ingress-controller --version
 
 By default, the controller watches Ingress objects from all namespaces. If you want to change this behavior,
 use the flag `--watch-namespace` or check the Helm chart value `controller.scope` to limit the controller to a single
-namespace.
+namespace. Although the use of this flag is not popular and also not encouraged so much, one important fact to note is that the secret containing the default-ssl-certificate needs to also be present in the watched namespace(s).
```

Comment:
```suggestion
namespace. Although the use of this flag is not popular, one important fact to note is that the secret containing the default-ssl-certificate needs to also be present in the watched namespace(s).
```

I'd not tell that. In the end this flag might make your deployment more secure.

Reply:
changed as sugested